### DISCUSSION TITLE
Background color Selection is not available

### DISCUSSION THREAD

**Discussion by davidakanjii** [royshil/obs-backgroundremoval#270] [General]
After I installed the Background Removal, the Effect Filters is not having the background colour option or is it that the latest background removers version I downloaded doesn't have it

This is the recent pictures  ![image](https://user-images.githubusercontent.com/47996814/233798123-bf29758b-4d56-4120-bf7d-b6e660d81f9e.png)

This is the old one: 
![image](https://user-images.githubusercontent.com/47996814/233798270-efbe6610-305f-4b40-baee-477a5ecd26ca.png)

**Comment by umireon** [comment]
We remove the background color option and the background color is fixed to be transparent for ease of configuration

**Reply by davidakanjii** [reply]
Thank you for your feedback. I have later confirmed it too. Well done on the great job

**Reply by kkfrosty** [reply]
How do you use a image or video as the background now as this walkthrough demostrates.  https://www.youtube.com/watch?v=0DD2RNcP0KQ  Only reason trying to figure out how to use this.

**Reply by umireon** [reply]
@kkfrosty I don't understand what you want us. We provide [our official tutorial video](https://www.youtube.com/watch?v=veqNEsMqEE0) and we cannot do anything about outdated videos made by people other than us.

**Reply by royshil** [reply]
@kkfrosty you're welcome to join my office hours stream and ask how to do it, and i will demonstrate
